src/webview/html/header.js

```diff
@@ -55,7 +55,13 @@ export default (
     `;
     } else if (headerStyle === 'full') {
       const streamName = streamNameOfStreamMessage(message);
-      const stream = subscriptions.find(x => x.name === streamName);
+      let stream = null;
+      for (const sub of subscriptions.values()) {
+        if (sub.name === streamName) {
```

Comment:
Let's add a comment here that says `TODO(#3339)` (with parens just like that -- a consistent format is helpful for grepping.) That's because this spot was and still is an instance of #3339, but in this form with an explicit loop it's not as easy to spot when searching for instances of that issue. So the TODO tag will help make sure we find it when we're closing in on finishing that issue.